MAUI/Maps/tooltip.md

```diff
@@ -10,6 +10,11 @@ documentation: ug
 # Tooltip in .NET MAUI Maps (SfMaps)
 A tooltip indicates the shape, bubble and marker information during the tap or hover interactions. This section demonstrates how to show tooltips for the shapes, bubbles, and markers in the maps and how to customize them.
 
+To learn more about our .NET MAUI Maps tooltip, you can check the below video.
```

Comment:
To learn more about the .NET MAUI Maps tooltip, you can check the following video.